session_4/blogspot/app/views/articles/show.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="jumbotron">
```

Comment:
You are enclosing everything in the jumbotron, which is why it looks "blurred"

Reply:
Tried Separately but didn't worked @abhishekkumar2718

Reply:
Also tried removing shadow according to some github issue chat, still the same